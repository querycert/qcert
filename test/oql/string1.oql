/* toString */
toString(new Customer( cid: 1, age: 32, name: "John" ))
