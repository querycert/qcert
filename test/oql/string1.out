"<Customer:{age->32, cid->1, name->John}>"